lib/palindrome_permutation.rb

```diff
@@ -1,5 +1,22 @@
 
 def palindrome_permutation?(string)
-
- 
-end # method
+  if string == ""
+    return true
+  end
+  hash = {}
+  string = string.split
+  string.each do |letter|
+    hash[letter] = hash[letter].nil? ? 1 : hash[letter] + 1
+  end
+  count = 0
+  negative_count = -1
+  until count == string.length
+  if string[count] != hash[negative_count]
+    return false
+  else
+    count += 1
+    negative_count -= 1
+  end
+  return true
+  end
```

Comment:
I'm not sure what this is doing.  

I suggest looping through the hash can counting the number of odd numbered letters.  A palindrome can have a maximum of 1 letter which occurs an odd number of times.